### PR TITLE
Updates to path

### PR DESCRIPTION
**Pull request by msalvaris**
- Updates to latest AML
- Adds makefile for local testing
- Adds get_auth for getting authorisation token
- Adds the rest of the notebooks to the pipeline

### CODE REVIEW

**Comment by msalvaris** [comment]
Good point. The plan is to get all the paths to follow this layour similar to https://github.com/Microsoft/Recommenders/ and the other BP repos
I will fix other issues :feelsgood:

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+clean: remove-notebook
+	conda remove --name batchscoringdl_aml -y --all
+	cd notebooks
+	./clean_up.sh
+	rm -rf aml_config
+	rm -rf aml_test_orangutan
+	rm -rf __pycache__
+	rm -rf .ipynb_checkpoints
+
+.PHONY: help test setup clean remove-notebook test-notebook1 test-notebook2 test-notebook3 test-notebook4 test-notebook5
```
Comment (jiata) [inline Makefile]: @msalvaris should `remove-notebook` be at the end?
I just ran through it all, but there is a remaining `notebooks/test.ipynb' in the repo that should've been deleted.

Comment (msalvaris) [reply Makefile]: @jiata I thought so as well but looking around it seems like the standard practice is to have someone run the clean at the end just incase people want to do something with the files. 
remove-notebook should remove test.ipynb

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		-k python3  
+
+remove-notebook: 
+	rm -f notebooks/test.ipynb
+
+clean: remove-notebook
+	conda remove --name batchscoringdl_aml -y --all
+	cd notebooks
+	./clean_up.sh
+	rm -rf aml_config
```
Comment (jiata) [inline Makefile]: these should be 
```
rm -rf notebooks/aml_config
rm -rf notebooks/aml_test_orangutan
rm -rf notebooks/.ipynb_checkpoints
```

Comment (jiata) [reply Makefile]: actually, not sure about this one. let me test

Comment (jiata) [reply Makefile]: yeah, didn't see the `cd` above

**Pull request merged**